cardano-api/src/Cardano/Api/MetaData.hs

```diff
@@ -0,0 +1,159 @@
+module Cardano.Api.MetaData
+  ( MetaDataJsonConversionError (..)
+  , jsonFromMetadata
+  , jsonToMetadata
+  , renderMetaDataJsonConversionError
+  ) where
+
+import           Cardano.Prelude hiding (MetaData)
+import           Prelude (String)
+
+import           Cardano.Api.Typed as Api
+
+import qualified Data.Aeson as Aeson
+import qualified Data.Attoparsec.Text as Atto
+import qualified Data.ByteString.Char8 as BS
+import qualified Data.ByteString.Base16 as Base16
+import qualified Data.HashMap.Strict as HashMap
+import qualified Data.List as List
+import qualified Data.Map.Strict as Map
```

Comment:
Any particular reason we use strict containers?

Reply:
We use strict pretty much thoughout the whole of the cardano code base.

We tend to use strict data and lazy functions as the default.